lib/docker-file.ts

```diff
@@ -10,15 +11,31 @@ async function getBaseImageName(targetFile?: string):
   }
 
   const contents = await readFile(targetFile);
+  const dockerfile = DockerfileParser.parse(contents);
+  const from = dockerfile.getFROMs().pop();
```

Comment:
Consider rename `from`. Maybe something like `fromContext`? :thinking:

Reply:
This is literally a string that is the final "FROM" line in a Dockerfile. I don't think it needs renaming, and I don't know what "context" is being added here.